config/src/main/resources/profiles/enterprise-private.toml

```diff
@@ -0,0 +1,6 @@
+sync-mode="X_SNAP"
+sync-min-peers=1
+remote-connections-limit-enabled=false
+tx-pool="SEQUENCED"
+tx-pool-no-local-priority=true
+tx-pool-limit-by-account-percentage=0.1220703125
```

Comment:
Probably worth adding `--rpc-http-max-active-connections=300` which we've found certain performance scenarios require